docs/source/changelog.rst

```diff
@@ -13,6 +13,35 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
+0.6.0 / 2019-08-14
+------------------
+
+New Features
+++++++++++++
+
+- (:pr:`85`, :pr:`87`) Msgpack is now supported as a serialization option for Models, and is also the default choice for
+  serialization with JSON fallback. This results in substantial speedups for both serialization and deserialization
+  actions and should be a transparent replacement for users within Elemental itself.
```

Comment:
Msgpack is a new serialization option for Models. Serialization defaults to msgpack when available (`conda install msgpack-python [-c conda-forge]`), falling back to JSON otherwise. This results ....